views/score.erb

```diff
@@ -0,0 +1,19 @@
+<h1> Score that Word </h1>
+
+<h3>Enter your scrabble word below to get the score for that word and each of its letters.</h3>
+
+<form action="" method="post" accept-charset="utf-8">
+  <label for="word">Word</label>
+  <input type="text" name="word" value="" id="word">
+  <input type="submit" value="Submit">
+</form>
+
+<h3>The word '<%= @word %>' has a score of <%= @word_score %>.</h3>
+
+<% (0...(@letters.length)).each do |index| %>
+  <% @letters[index] %>
```

Comment:
It doesn't look like this line and the one below are used for anything, since they are displayed on line 16. Were these here for debugging purposes or something else?